distribution/changelog.txt

```diff
@@ -53,6 +53,7 @@
 - Fix: [#8469] Crash modifying colour on hacked rides.
 - Fix: [#8508] Underground roto-drop is not going up.
 - Fix: [#8555] Multiplayer window text limits are not computed properly.
+- Fix: [#8591] Run game loop at consistent tick rate of 40hz.
```

Comment:
This should list the problem that was fixed, rather than the solution.
```suggestion
- Fix: [#8591] Game loop does not run at a consistent tick rate of 40 Hz.
```